client/src/components/Input/AutoSize.js

```diff
@@ -1,15 +1,42 @@
 import React from "react";
+import styled from "styled-components";
 import TextArea from "./TextArea";
+import SvgIcon from "components/Icon/SvgIcon";
+import send from "assets/icons/send.svg";
+import { Button } from "antd";
+import { mq, theme } from "constants/theme";
+const { white } = theme.colors;
+
+const SendButton = styled(Button)`
+  align-items: center;
+  border: none;
+  color: ${white};
+  cursor: pointer;
+  display: flex;
+  float: right;
+  height: 4.5rem;
+  justify-content: center;
+
+  @media only screen and (max-width: ${mq.phone.narrow.maxWidth}) {
+    height: 5rem;
+    padding: 1rem 1rem;
+  }
+`;
 
 const AutoSize = ({ placeholder, value, onChange, onPressEnter }) => {
   return (
-    <TextArea
-      autoSize
-      value={value}
-      onChange={onChange}
-      onPressEnter={onPressEnter}
-      placeholder={placeholder}
-    />
+    <div>
```

Comment:
Do we need this`<div>` here?

Reply:
we can only return one element in the return section, since I have two elements I needed to wrap them in a div to count as one

Reply:
For these cases, you can use fragments instead of <div></div>: https://reactjs.org/docs/fragments.html  short syntax: `<></>`

So insteand of `<div></div>` use `<></>`

Reply:
thats right :) I'll change is!